### PR TITLE
Update 0x1004E947366A632d458D4970c70F6b1F6A0DE710

### PR DESCRIPTION
**Pull request by axeldlh**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- [ ]  Add a new file (or update existing file) into the `set-details` or `set-details-polygon` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [ ] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [ ] If adding brand new token, make sure to add it to the `set.tokenlist.json` or `set.polygon.tokenlist.json` as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [ ] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [ ] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [ ] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

[INSERT SIGNATURE HERE]
0xe10cb6fda8d1ba6b93f85fb163422ff3d2e01787e77357d691e35cbd50cb62a173b0b53e460d38a69e2a02298e56bed299db26e88d8e1a9deb0a2f4075d150f91b

### CODE REVIEW

**Comment by ADWilkinson** [comment]
I can't find your Set in in setdetails.json (searched by 0x1004E947366A632d458D4970c70F6b1F6A0DE710) in the root directory, add it in there when you get a chance

**Comment by axeldlh** [comment]
Yes, Updated set.tokenlist.json

File: set-details/0x1004E947366A632d458D4970c70F6b1F6A0DE710.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Eldar Venture Fund",
+    "setSymbol": "EVF",
+    "setDecimals": 18,
+    "setDescription": "Eldar Venture is our early-stage token fund, seeking to invest in recent crypto projects that have a promising use case and huge market potential, in order to provide VC-like returns.\n\nEldar Venture selects early crypto projects that have an exceptional founding team, an innovative working product and significative early growth. Most of the tokens selected are also backed by a set of prominent crypto VC firms.\n\nYou can learn more about our investment process at: https://www.eldarcap.com/investment-process",
```
Comment (justinkchen) [inline set-details/0x1004E947366A632d458D4970c70F6b1F6A0DE710.setdetails.json:7]: missing a setIcon

**Comment by justinkchen** [comment]
the eldercapital.png should not be at the root level, should be in the assets folder

File: set-details/0x1004E947366A632d458D4970c70F6b1F6A0DE710.setdetails.json (hunk 1 above)
Comment (axeldlh) [reply set-details/0x1004E947366A632d458D4970c70F6b1F6A0DE710.setdetails.json:7]: The token does not have a specific icon yet. This worked for my two other sets.

**Comment by axeldlh** [comment]
Yes, I have deleted it from the root. IDK why it was there, it was also in the assets folder. At least I can see it here on my part. I hope everything is fine now.

**Comment by axeldlh** [comment]
for the file in asset I have just reduced the png size, so that is why it appears as deleted on the review but it is present.

**Review by justinkchen — APPROVED** [review]

**Pull request merged**